jenkinsci/jenkins
# Display a notice when there are plugins installed or updates available

Followup to #8186. Adds a notice to the Installed plugins page(s) as well as to the updates page to tell the user when there are no plugins installed/updates available, this is rather than displaying the table. Also disables the search bar.

<img width="1089" alt="image" src="https://github.com/jenkinsci/jenkins/assets/43062514/bdaee44a-266a-41ab-9bd1-33629dbc1f11">

<img width="1087" alt="image" src="https://github.com/jenkinsci/jenkins/assets/43062514/e7c43df4-781c-4680-9dcd-8a57939e237e">

<img width="1077" alt="image" src="https://github.com/jenkinsci/jenkins/assets/43062514/8b22b9ac-74eb-4d21-8052-d2c256b8614d">

### Testing done

* Notice shows when no plugins are installed
* Notice hides when plugins are installed
* Notice shows when no updates are available
* Notice hides when updates are available

### Proposed changelog entries

- Display a notice when plugin updates are available or when there are no plugins installed.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
```

### Desired reviewers

@jenkinsci/sig-ux 



Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
```


<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8208"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
